input/core/build.gradle.kts

```diff
@@ -87,18 +88,29 @@ kotlin {
 //            implementation(libs.krono.kotlinx)
 //            implementation(coil.compose)
 //            implementation(coil.network.ktor)
+            api(compose.components.resources)
         }
 
         commonTest.dependencies {
             implementation(libs.kommander.coroutines)
             implementation(compose.uiTest)
         }
 
+        androidMain.dependencies {
+            implementation("androidx.activity:activity-ktx:1.10.1")?.because("We need ComponentActivity for android file picker")
+        }
+        jvmMain.dependencies {
+            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-swing:1.7.3")
```

Comment:
Why are we adding this dependency? Are you looking for `Dispatchers.Main` in Desktop? I don't think it is need, is it?